rotkehlchen/tests/unit/test_utils.py

```diff
@@ -58,6 +59,34 @@ def test_process_result():
     json.dumps(process_result(d))
 
 
+def test_process_result_recursively_processes_tuples():
+    nested_tuple = {
+        'tuple': (
+            FVal('1.5'),
+            {
+                'nested': (
+                    FVal('3.2'),
+                    Asset('ETH'),
```

Comment:
something minor but you already have `A_USDC`. It is possible to avoid the Asset import